gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Nov 07 18:46:28 KST 2017
```

Comment:
Sure.

Reply:
I did some search on stackoverflow. Some folks said it should be added to the git repo. What do you say?

> It should be included in your git repo.
>By using it with gradlew, this will ensure all your developers (and your build servers!) always use the same version of Gradle. This way you know that every one is building the same exact way.

https://stackoverflow.com/questions/21762180/should-you-include-or-ignore-gradle-wrapper-properties

Reply:
Yes, you are correct.
I got confused.